custom_components/kamstrup_403/sensor.py

```diff
@@ -73,6 +73,20 @@
         icon="mdi:water",
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    SensorEntityDescription(
+        key="84",  # 0x0054
+        name="WaterMeter1",
+        icon="mdi:water",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
+    ),
+    SensorEntityDescription(
+        key="85",  # 0x0055
+        name="WaterMeter2",
+        icon="mdi:water",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        entity_registry_enabled_default=False,
+    ),
```

Comment:
I'm not happy with the current naming convention of the sensors (see #76) For new sensors we don't need to worry about backward compatibility, so we should do it right.

```suggestion
    SensorEntityDescription(
        key="84",  # 0x0054
        name="Watermeter ",
        icon="mdi:water",
        state_class=SensorStateClass.TOTAL_INCREASING,
        entity_registry_enabled_default=False,
    ),
    SensorEntityDescription(
        key="85",  # 0x0055
        name="Watermeter",
        icon="mdi:water",
        state_class=SensorStateClass.TOTAL_INCREASING,
        entity_registry_enabled_default=False,
    ),
```